diff --git a/node_modules/discord-akairo/src/struct/commands/CommandHandler.js b/node_modules/discord-akairo/src/struct/commands/CommandHandler.js
index fdae608..5bf0043 100644
--- a/node_modules/discord-akairo/src/struct/commands/CommandHandler.js
+++ b/node_modules/discord-akairo/src/struct/commands/CommandHandler.js
@@ -216,8 +216,6 @@ class CommandHandler extends AkairoHandler {
          * @name CommandHandler#modules
          * @type {Collection<string, Command>}
          */
-
-        this.setup();
     }
 
     setup() {
diff --git a/node_modules/discord-akairo/src/util/Constants.js b/node_modules/discord-akairo/src/util/Constants.js
index bacd4e1..4d4810e 100644
--- a/node_modules/discord-akairo/src/util/Constants.js
+++ b/node_modules/discord-akairo/src/util/Constants.js
@@ -79,7 +79,7 @@ module.exports = {
     BuiltInReasons: {
         CLIENT: 'client',
         BOT: 'bot',
-        OWNER: 'owner',
+        OWNER: 'Only jacany#0001 can use this command!',
         GUILD: 'guild',
         DM: 'dm'
     }
