diff --git a/node_modules/discord-akairo/src/index.d.ts b/node_modules/discord-akairo/src/index.d.ts
index cb4d20a..690e326 100644
--- a/node_modules/discord-akairo/src/index.d.ts
+++ b/node_modules/discord-akairo/src/index.d.ts
@@ -28,6 +28,11 @@ declare module 'discord-akairo' {
         public util: ClientUtil;
 
         public isOwner(user: UserResolvable): boolean;
+
+        public config: any
+        public settings: SequelizeProvider;
+        public shoukaku: any;
+        public queue: any;
     }
 
     export class AkairoHandler extends EventEmitter {
