diff --git a/node_modules/discord-paginationembed/bin/base/index.js b/node_modules/discord-paginationembed/bin/base/index.js
index cc6bd8a..cdc57a4 100644
--- a/node_modules/discord-paginationembed/bin/base/index.js
+++ b/node_modules/discord-paginationembed/bin/base/index.js
@@ -73,7 +73,7 @@ exports.PaginationEmbed = class extends t.EventEmitter {
   }
   setDisabledNavigationEmojis(t) {
     if (!Array.isArray(t)) throw new TypeError("Cannot invoke PaginationEmbed class without a valid array.");
-    const e = [], i = [];
+    const e = [], i = ["jump"];
     for (const s of t) [ "back", "jump", "forward", "delete", "all" ].includes(s) ? i.push(s) : e.push(s);
     if (e.length) throw new TypeError(`Cannot invoke PaginationEmbed class with invalid navigation emoji(s): ${e.join(", ")}.`);
     return this.disabledNavigationEmojis = i, this;
@@ -125,7 +125,7 @@ exports.PaginationEmbed = class extends t.EventEmitter {
   async _checkPermissions() {
     const t = this.channel;
     if (t.guild) {
-      const e = t.permissionsFor(t.client.user).missing([ "ADD_REACTIONS", "MANAGE_MESSAGES", "EMBED_LINKS", "VIEW_CHANNEL", "SEND_MESSAGES" ]);
+      const e = t.permissionsFor(t.client.user).missing([ "ADD_REACTIONS", "EMBED_LINKS", "VIEW_CHANNEL", "SEND_MESSAGES" ]);
       if (e.length) throw new Error(`Cannot invoke PaginationEmbed class without required permissions: ${e.join(", ")}`);
     }
     return !0;
