diff --git a/node_modules/discord-akairo/src/index.d.ts b/node_modules/discord-akairo/src/index.d.ts
index cb4d20a..5d12fb8 100644
--- a/node_modules/discord-akairo/src/index.d.ts
+++ b/node_modules/discord-akairo/src/index.d.ts
@@ -28,6 +28,12 @@ declare module 'discord-akairo' {
         public util: ClientUtil;
 
         public isOwner(user: UserResolvable): boolean;
+
+        public config: any
+        public settings: SequelizeProvider;
+        public shoukaku: any;
+        public queue: any;
+        public log: bunyan;
     }
 
     export class AkairoHandler extends EventEmitter {
@@ -169,6 +175,7 @@ declare module 'discord-akairo' {
         public ignoreCooldown?: Snowflake | Snowflake[] | IgnoreCheckPredicate;
         public ignorePermissions?: Snowflake | Snowflake[] | IgnoreCheckPredicate;
         public ownerOnly: boolean;
+        public modOnly: boolean;
         public prefix?: string | string[] | PrefixSupplier;
         public ratelimit: number;
         public regex: RegExp | RegexSupplier;
@@ -561,6 +568,7 @@ declare module 'discord-akairo' {
         lock?: KeySupplier | 'guild' | 'channel' | 'user';
         optionFlags?: string[];
         ownerOnly?: boolean;
+        modOnly?: boolean;
         prefix?: string | string[] | PrefixSupplier;
         ratelimit?: number;
         regex?: RegExp | RegexSupplier;
diff --git a/node_modules/discord-akairo/src/struct/commands/Command.js b/node_modules/discord-akairo/src/struct/commands/Command.js
index 1003663..3ecc37a 100644
--- a/node_modules/discord-akairo/src/struct/commands/Command.js
+++ b/node_modules/discord-akairo/src/struct/commands/Command.js
@@ -21,6 +21,7 @@ class Command extends AkairoModule {
             separator,
             channel = null,
             ownerOnly = false,
+            modOnly = false,
             editable = true,
             typing = false,
             cooldown = null,
@@ -74,6 +75,12 @@ class Command extends AkairoModule {
          */
         this.ownerOnly = Boolean(ownerOnly);
 
+        /**
+         * Usable only by the discord mods.
+         * @type {boolean}
+         */
+        this.modOnly = Boolean(modOnly);
+
         /**
          * Whether or not this command can be ran by an edit.
          * @type {boolean}
@@ -248,6 +255,7 @@ module.exports = Command;
  * @prop {string[]} [optionFlags=[]] - Option flags to use when using an ArgumentGenerator.
  * @prop {string} [channel] - Restricts channel to either 'guild' or 'dm'.
  * @prop {boolean} [ownerOnly=false] - Whether or not to allow client owner(s) only.
+ * @prop {boolean} [modOnly=false] - Whether or not to allow mods only.
  * @prop {boolean} [typing=false] - Whether or not to type in channel during execution.
  * @prop {boolean} [editable=true] - Whether or not message edits will run this command.
  * @prop {number} [cooldown] - The command cooldown in milliseconds.
