diff --git a/node_modules/discord-akairo/src/struct/commands/CommandHandler.js b/node_modules/discord-akairo/src/struct/commands/CommandHandler.js
index 818ddc4..ec2899d 100755
--- a/node_modules/discord-akairo/src/struct/commands/CommandHandler.js
+++ b/node_modules/discord-akairo/src/struct/commands/CommandHandler.js
@@ -216,26 +216,6 @@ class CommandHandler extends AkairoHandler {
          * @name CommandHandler#modules
          * @type {Collection<string, Command>}
          */
-
-        this.setup();
-    }
-
-    setup() {
-        this.client.once('ready', () => {
-            this.client.on('message', async m => {
-                if (m.partial) await m.fetch();
-                this.handle(m);
-            });
-
-            if (this.handleEdits) {
-                this.client.on('messageUpdate', async (o, m) => {
-                    if (o.partial) await o.fetch();
-                    if (m.partial) await m.fetch();
-                    if (o.content === m.content) return;
-                    if (this.handleEdits) this.handle(m);
-                });
-            }
-        });
     }
 
     /**
diff --git a/node_modules/discord-akairo/src/util/Constants.js b/node_modules/discord-akairo/src/util/Constants.js
index bacd4e1..4d4810e 100755
--- a/node_modules/discord-akairo/src/util/Constants.js
+++ b/node_modules/discord-akairo/src/util/Constants.js
@@ -79,7 +79,7 @@ module.exports = {
     BuiltInReasons: {
         CLIENT: 'client',
         BOT: 'bot',
-        OWNER: 'owner',
+        OWNER: 'Only jacany#0001 can use this command!',
         GUILD: 'guild',
         DM: 'dm'
     }
