diff --git a/node_modules/lavasfy/dist/structures/Resolver.js b/node_modules/lavasfy/dist/structures/Resolver.js
index 73723a6..bd573d0 100644
--- a/node_modules/lavasfy/dist/structures/Resolver.js
+++ b/node_modules/lavasfy/dist/structures/Resolver.js
@@ -30,6 +30,7 @@ class Resolver {
             playlistInfo: {
                 name: album === null || album === void 0 ? void 0 : album.name
             },
+            spotifyMetadata: album,
             tracks: album
                 ? (await Promise.all(album.tracks.items.map(x => this.resolve(x)))).filter(Boolean)
                 : []
@@ -47,6 +48,7 @@ class Resolver {
             playlistInfo: {
                 name: playlist === null || playlist === void 0 ? void 0 : playlist.name
             },
+            spotifyMetadata: playlist,
             tracks: (await Promise.all(playlistTracks.map(x => x.track && this.resolve(x.track)))).filter(Boolean)
         };
     }
@@ -60,6 +62,7 @@ class Resolver {
         return {
             loadType: lavaTrack ? "TRACK_LOADED" : "NO_MATCHES",
             playlistInfo: {},
+            spotifyMetadata: track,
             tracks: lavaTrack ? [lavaTrack] : []
         };
     }
diff --git a/node_modules/lavasfy/dist/typings/Lavalink/index.d.ts b/node_modules/lavasfy/dist/typings/Lavalink/index.d.ts
index 7cb8634..526c9c3 100644
--- a/node_modules/lavasfy/dist/typings/Lavalink/index.d.ts
+++ b/node_modules/lavasfy/dist/typings/Lavalink/index.d.ts
@@ -24,6 +24,7 @@ export interface LavalinkTrackResponse {
         selectedTrack?: number;
     };
     tracks: LavalinkTrack[];
+    spotifyMetadata: any;
     exception?: {
         message: string;
         severity: string;
